solidity/random-beacon/test/RandomBeacon.Relay.test.ts

```diff
@@ -277,7 +277,7 @@ describe("RandomBeacon - Relay", () => {
     })
   })
 
-  describe("submitRelayEntry", () => {
+  describe("submitRelayEntry before the soft timeout - happy path", () => {
```

Comment:
I was confused with the test naming. I'm wondering whether it wouldn't be better to name the tests for overlapping function names like:
```
describe("submitRelayEntry(bytes)", () => {
```
and
```
describe("submitRelayEntry(bytes,uint32[])", () => {
```

It would be consistent with other tests as we always use the function name in `describe` message.

Reply:
I would definitely rename to include the signature just like @nkuba suggests.

Reply:
dfe4320